### PR TITLE
Dyanmic thumbnails generated according to the title of the video , notion , folder(current week)

### LINKED ISSUE #790: bug: Folder , Video , Notion doesnot have dynamic content  & thumbnail.
**Describe the bug**
Basically when we open any course there we can see if it is a folder then the thumbnail and the content inside it static for all ,
It affects user experience while learning / exploring courses.

**To Reproduce**
1.Just open any course there you will see the folder is like simple folder image  and for video it is just a video image , and same as for notion notes.

**Expected behavior**
It would be better if we have a dynamic content where the text inside image would be title of the content like if it is folder then it should be week 1 | week 2 like that , when it  notion then it should be next auth | some notes title and when it is video the title of the video will be displayed there .

**Screenshots or GIFs**
before_update

**Info (please complete the following information):**
 - Browser [chrome]
 - Version [22]

### PR DESCRIPTION
**Pull request by kethesainikhil**
### PR Fixes:
- 1 Introuduced the dynamic content generated (card component) so it changes the content inside based on the title of the  folder , video , notion accordingly.
- 2 To not to disturb the design of the previous static images I have designed the same card like template so it changes the content inside the card and also the icons will be dynamically changed.

Resolves #790 .
Here are the proofs of how this works and my design , I am providing both the screenshots as well as screencast so in screencast you can able to see the responsiveness of the card component
screenshot of how video and notion looks like
Screenshot from 2024-06-15 22-41-21
screenshot of how folder looks like
Screenshot from 2024-06-15 22-56-20
screencast of all together with mobile responsiveness has been shown here
[Screencast from 2024-06-16 00-08-27.webm](https://github.com/code100x/cms/assets/70809367/740d8bd4-4de6-4a11-a482-ad0ced287ab3)

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by kethesainikhil** [comment]
@hkirat  could you please review this as it makes user experience very dynamic and able to crawl over through the content very easily for users.

File: src/components/Certificate.tsx [hunk 1]
```diff
@@ -95,11 +95,7 @@ export const CertificateComponent = ({
         </Button>
         <div className="flex items-center justify-center">
           <Button
-<<<<<<< tailwind-prettier
```
Comment (hkirat) [inline src/components/Certificate.tsx:98]: where did this get in?
@siinghd

Comment (kethesainikhil) [reply src/components/Certificate.tsx:98]: It was a merge conflit so I have allowed a incoming one

**Comment by kethesainikhil** [comment]
> very cool PR

Thanks @hkirat

File: src/components/Certificate.tsx (hunk 1 above)
Comment (siinghd) [reply src/components/Certificate.tsx:98]: @hkirat tailwind prettier was  here https://github.com/code100x/cms/pull/770/

**Comment by siinghd** [comment]
testing it RIght now

**Comment by siinghd** [comment]
Merging, you will get the bounty for sure

**Comment by kethesainikhil** [comment]
> Merging, you will get the bounty for sure

Thank you @siinghd

**Pull request merged**